Subdivision_method_3/include/CGAL/Subdivision_method_3/subdivision_masks_3.h

```diff
@@ -571,12 +571,12 @@ class Sqrt3_mask_3 : public Linear_mask_3<PolygonMesh, VertexPointMap> {
   /// computes the \f$ \sqrt{3}\f$ vertex-point `pt` of the vertex `vd`.
   void vertex_node(vertex_descriptor vertex, Point& pt) {
     Halfedge_around_target_circulator<Mesh> vcir(vertex, *(this->pmesh));
-    const typename boost::graph_traits<Mesh>::degree_size_type n = degree(vertex, *(this->pmesh));
+    const int n = degree(vertex, *(this->pmesh));
```

Comment:
You will probably have warnings. Use that instead:
```suggestion
    const int n = static_cast<int>(degree(vertex, *(this->pmesh)));
```

Reply:
Indeed.  The corresponding GCC warning option isn't enabled by `-Wall` or `-Wextra`, but it seems to be enabled by default on MSVC.  Fixed; thanks!